pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1661,3 +1726,48 @@ func (ctrl *PersistentVolumeController) findDeletablePlugin(volume *v1.Persisten
 	}
 	return plugin, nil
 }
+
+// obtain provisioner/deleter name for a volume
+func (ctrl *PersistentVolumeController) getProvisionerNameFromVolume(volume *v1.PersistentVolume) string {
+	plugin, err := ctrl.findDeletablePlugin(volume)
+	if err != nil {
+		return "N/A"
```

Comment:
Local PV will end up with "N/A" as the plugin since it's not dynamically provisioned and the plugin itself doesn't implement delete.

Should we let it fall through instead of returning "N/A" here?